### DISCUSSION TITLE
What version of Bootstrap is used in LeptonXLite?

### DISCUSSION THREAD

**Discussion by xorinzor** [abpframework/abp#16436] [Q&A] [answered]
For UI development it'd be useful to know what version of bootstrap is used by LeptonXLite, otherwise we'll be looking at potentially the wrong documentation for bootstrap.

Was able to find a dependency named @abp/boostrap but that package version doesn't match bootstrap's versioning.
Documentation doesn't seem to provide any information on this either, nor in any of the package readme's.

**Comment by EngincanV** [comment]
LeptonX Lite Theme is built on Bootstrap 5.

**Reply by xorinzor** [reply]
Which version of Bootstrap 5?

**Reply by EngincanV** [reply, marked as answer]
> Which version of Bootstrap 5?

`^5.1.3` -> it corresponds to *5.2.3*, i guess.

**Reply by xorinzor** [reply]
Thanks!
